vulyk/assets.py

```diff
@@ -1,9 +1,32 @@
-# coding=utf-8
+# -*- coding: utf-8 -*-
+from os import curdir, listdir
+from os.path import isfile, join
+
 from flask.ext.assets import Environment, Bundle
 
 
 def init(app):
     assets = Environment(app)
+
+    enabled_tasks = app.config.get("ENABLED_TASKS", {})
+    for plugin in enabled_tasks:
+        plugin = plugin.replace('.', '/')
+        js_path = '{cur_dir}/{plugin_name}/static/scripts'.format(
```

Comment:
I would rather do import (as dot-path path to plugin might be something like .mypluggin or something).

Then you can use os.path.basedir(plugin.__file__) or something